snapshot/lib/snapshot/page.html.erb

```diff
@@ -64,23 +90,48 @@
       }
     </style>
   </head>
-  <body><% image_counter = 0 %><% @data.each do |language, content| %>
-    <h1 id="<%= language %>" class="language"><%= language %></h1>
-    <hr>
-    <table><% content.each do |device_name, screens| %>
-      <tr>
-        <th colspan="<%= screens.count %>">
-          <a id="<%= language %>-<%= device_name %>" class="deviceName" href="#<%= language %>-<%= device_name %>"><%= device_name %></a>
-        </th>
-      </tr>
-      <tr><% screens.each do |screen_path| %><% next if screen_path.include?"_framed.png" %>
-        <td><% image_counter += 1 %>
-          <a href="<%= screen_path %>" target="_blank" class="screenshotLink">
-            <img class="screenshot" src="<%= screen_path %>" style="width: 100%;" alt="<%= language %> <%= device_name %>" data-counter="<%= image_counter %>">
-          </a>
-        </td><% end %>
-      </tr><% end %>
-    </table><% end %>
+  <body>
+    <div id="sortMenu">
+      <button id="defaultTab" class="tabLink" onclick="openTab(event, 'byLanguage')">By Language</button>
+      <button class="tabLink" onclick="openTab(event, 'byScreen')">By Screen</button>
+    </div>
+    <div id="byLanguage" class="tabContent"><h1 class="tabTitle">By Language:</h1><% image_counter = 0 %><% @data_by_language.each do |language, content| %>
+      <h1 id="<%= language %>"><%= language %></h1>
```

Comment:
I know this doesn't really matter here, but could we bump those down to h2 so the semantics are more correct? Thanks.

Reply:
For the language, you mean?

Reply:
yep. but actually, I'm fine either way.

Reply:
I can fix that. Makes sense.